packages/vite/package.json

```diff
@@ -99,7 +100,7 @@
     "launch-editor-middleware": "^2.2.1",
     "magic-string": "^0.25.7",
     "mime": "^3.0.0",
-    "minimatch": "^3.0.4",
+    "micromatch": "^4.0.4",
```

Comment:
I think we want to sort deps alphabetically like `pnpm add` is doing it out of the box.
So please swap mime with micromatch

Reply:
thanks, done 👍